eq-author/src/components/RichTextEditor/RichTextEditor.test.js

```diff
@@ -549,4 +549,48 @@ describe("components/RichTextEditor", function() {
       );
     });
   });
+
+  describe("validation", () => {
+    it("should set input as valid if no validation message is passed as parameter", () => {
+      expect(wrapper.find("RichTextEditor__Input").prop("invalid")).toEqual(
+        false
+      );
+    });
+
+    it("should not display validation message is passed as parameter", () => {
```

Comment:
I dont get what this is testing. Could description be a little better?

Reply:
I've changed it to be more descriptive.